sections/main-collection-banner.liquid

```diff
@@ -36,7 +36,7 @@
           "
           src="{{ collection.image | image_url: width: 750 }}"
           sizes="(min-width: {{ settings.page_width }}px) {{ settings.page_width | minus: 100 | divided_by: 2 }}px, (min-width: 750px) calc(50vw - 130px), calc(50vw - 55px)"
-          alt=""
+          alt="{{ collection.image.alt | escape }}"
```

Comment:
I just did a quick test and it indeed defaults to the collection title, so something is going on at the admin level.

I agree that this is the ideal behavior: `if no alt text is added it will be empty and if not we will use what you've added as an alt text`